.pnp.cjs

```diff
@@ -8314,7 +8314,7 @@ function $$SETUP_STATE(hydrateRuntimeState, basePath) {
       ]],\
       ["@surma/rollup-plugin-off-main-thread", [\
         ["npm:2.2.3", {\
-          "packageLocation": "./.yarn/unplugged/@surma-rollup-plugin-off-main-thread-npm-2.2.3-1f57d3eded/node_modules/@surma/rollup-plugin-off-main-thread/",\
+          "packageLocation": "./.yarn/cache/@surma-rollup-plugin-off-main-thread-npm-2.2.3-1f57d3eded-2c02134944.zip/node_modules/@surma/rollup-plugin-off-main-thread/",\
```

Comment:
If we change this back to how it once was, I'm wondering if `yarn` is going to try to revert it back to unplugged later. I'm not sure whether unplugged or cache, is correct, but I've found that deleting `.yarn/cache` at least makes `yarn` stop making changes every time you run `yarn`.

Reply:
After deleting ``.yarn/cache` and running `yarn` again, it made this change again.

Reply:
Since this PR merged, after pulling from `main`, `yarn` wants to revert it back to `unplugged` on my machine. Is it doing that for you?

Reply:
No, yarn doesn't revert it back. weird.